app/controllers/content_controller.rb

```diff
@@ -10,4 +10,9 @@ def update_links
   rescue GdsApi::HTTPConflict
     redirect_to :back, danger: "Somebody changed the tags before you could. Your changes have not been saved."
   end
+
+  def slug
```

Comment:
Would this be better in the `LookupController`? We could re-use the `find_by_slug` method.